src/netius/test/sh/base.py

```
@@ -0,0 +1,97 @@
+""" The copyright for the module """
+
+__license__ = "Apache License, Version 2.0"
+""" The license for the module """
+
+import sys
+import unittest
+
+import netius.sh.base
+
+try:
+    import unittest.mock as mock
```

Pylint (4.0.7)

[refactor] Use 'from unittest import mock' instead

(R0402)